bot/cogs/information.py

```diff
@@ -32,20 +33,18 @@ def __init__(self, bot: Bot):
         # Sort the roles alphabetically and remove the @everyone role
         roles = sorted(ctx.guild.roles[1:], key=lambda role: role.name)
 
-        # Build a string
-        role_string = ""
+        # Build a list
+        role_list = []
         for role in roles:
-            role_string += f"`{role.id}` - {role.mention}\n"
+            role_list.append(f"`{role.id}` - {role.mention}")
 
         # Build an embed
         embed = Embed(
-            title="Role information",
-            colour=Colour.blurple(),
-            description=role_string
+            title=f"Role information (Total {len(roles)} roles)",
```

Comment:
Now it will never happen that we have 0 or 1 role. and adding a role/roles pluralization check will just mess up the unit test. but if it is ever needed you could do something like

```python
f"{'s'*0 if True else 's'}"
```
Where True here is the result of a check for len(roles) being 1 or lower.

I think this PR is fine without this addition, this comment is just meant as a reference.

Reply:
You can further shorten this by doing
```py
's' * (number_of_roles > 1)
```
since `'s' * 0` is `''`